utilities_general.py

```
@@ -1,3 +1,1903 @@
+            novabusca =  f"SELECT  H.hash, trim(replace(snippet(documents, -1, '', '', '...', 200), char(13), ' ')), H.id_pdf, H.page, H.x0, H.y0, H.x1, H.y1 \
+                FROM db1.documents C INNER JOIN \
+                db2.Anexo_Eletronico_Pdf_Hashes H on (C.hash = H.hash) where C.content MATCH ? ORDER BY 1"          
+        if(tipobusca=="LIKE"):  
+            novabusca =  f"SELECT  H.hash, trim(replace(snippet(documents, -1, '', '', '...', 200), char(13), ' ')), H.id_pdf, H.page, H.x0, H.y0, H.x1, H.y1 FROM db1.documents C \
+                INNER JOIN db2.Anexo_Eletronico_Pdf_Hashes H \
+                on (C.hash = H.hash) where C.content like :termo ESCAPE :escape ORDER BY 1"        
+        if(tipobusca=="REGEX"):  
+            sqliteconn.create_function("REGEXP", 2, re_fn)
+            novabusca =  f"SELECT H.hash, trim(replace(snippet(documents, -1, '', '', '...', 200), char(13), ' ')), H.id_pdf, H.page, H.x0, H.y0, H.x1, H.y1 \
+                FROM db2.documents INNER JOIN db2.Anexo_Eletronico_Pdf_Hashes H \
+                on (C.hash = H.hash) where C.content REGEXP :regex ORDER BY 1"
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)